doc/Makefile

```diff
@@ -37,6 +37,8 @@ api-clean:
 api:
 	@mkdir -p reference
 	$(PYTHON) tools/build_modref_templates.py dipy reference
+	@mkdir -p reference_cmd
+	$(PYTHON) tools/docgen_cmd.py dipy reference_cmd
```

Comment:
No strong opinion on this, but do others think this might be better moved in a separate section, called something like ```command-line-api```.